src/core/Observable_stat.cpp

```diff
@@ -32,28 +32,15 @@
 
 extern boost::mpi::communicator comm_cart;
```

Comment:
Sorry I missed that in the review of the last PR. Please don't use extern definitions, this makes tool usage impossible and can lead to very obscure errors, e.g. if the original variable is renamed or it's type is changed. Also the dependency (on `communication.hpp`) should be explicitly stated.

Reply:
That's an oversight from my part, this extern is unnecessary with the proper include. Thanks!